packages/lesswrong/components/common/Header.tsx

```diff
@@ -289,6 +504,134 @@ const Header = ({
       <UsersMenu />
     </AnalyticsContext>
   </div>
+  
+  // the items on the right-hand side (search, notifications, user menu, login/sign up buttons)
+  const rightHeaderItemsNode = <div className={classes.rightHeaderItems}>
+    <NoSSR onSSR={<div className={classes.searchSSRStandin} />} >
+      <SearchBar onSetIsActive={setSearchOpen} searchResultsArea={searchResultsArea} />
+    </NoSSR>
+    {!isEAForum && usersMenuNode}
+    {!currentUser && <UsersAccountMenu />}
+    {currentUser && !currentUser.usernameUnset && <KarmaChangeNotifier
+      currentUser={currentUser}
+      className={(isEAForum && searchOpen) ? classes.hideXsDown : undefined}
+    />}
+    {currentUser && !currentUser.usernameUnset && <NotificationsMenuButton
+      unreadNotifications={unreadNotifications}
+      toggle={handleNotificationToggle}
+      open={notificationOpen}
+      className={(isEAForum && searchOpen) ? classes.hideXsDown : undefined}
+    />}
+    {isEAForum && usersMenuNode}
+  </div>
+  
+  // the left side nav menu
+  const navigationDrawer = <NavigationDrawer
+    open={navigationOpen}
+    handleOpen={() => setNavigationOpen(true)}
+    handleClose={() => setNavigationOpen(false)}
+    toc={toc?.sectionData ?? null}
+  />
+  
+  // the right side notifications menu
+  const notificationsMenu = currentUser && <NotificationsMenu
+    unreadPrivateMessages={unreadPrivateMessages}
+    open={notificationOpen}
+    hasOpened={notificationHasOpened}
+    setIsOpen={handleSetNotificationDrawerOpen}
+  />
```

Comment:
Pulled out a few pieces that are unchanged between the standard header and the giving season one